### PR TITLE
8296646: com/sun/jdi/JdbLastErrorTest.java test failure

### PR DESCRIPTION
**Pull request by kevinjwalls**
This test fails sometimes with ZGC. It is testing using Panama to access Windows' last error value, and can of course be interrupted by VM work that changes the value, but it does generally work. Skipping the test with ZGC is reasonable as you don't really _have_ to step through this kind of Panama code with ZGC.

More rare is a failure where "The application exited" is not observed.  This test is not here to test the exit mode, and logging that failure avoids telling you whether the correct lastError value was seen.  Test should prioritise saying if the correct last error value was observed, and should check for the exception being thrown when the correct value is not seen.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8296646](https://bugs.openjdk.org/browse/JDK-8296646): com/sun/jdi/JdbLastErrorTest.java  test failure

### Reviewers
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**) ⚠️ Review applies to [e37f1e73](https://git.openjdk.org/jdk/pull/12441/files/e37f1e73973df7aa58185163567a70528458d65b)
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**) ⚠️ Review applies to [e37f1e73](https://git.openjdk.org/jdk/pull/12441/files/e37f1e73973df7aa58185163567a70528458d65b)
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**) ⚠️ Review applies to [e37f1e73](https://git.openjdk.org/jdk/pull/12441/files/e37f1e73973df7aa58185163567a70528458d65b)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12441/head:pull/12441` \
`$ git checkout pull/12441`

Update a local copy of the PR: \
`$ git checkout pull/12441` \
`$ git pull https://git.openjdk.org/jdk pull/12441/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12441`

View PR using the GUI difftool: \
`$ git pr show -t 12441`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12441.diff">https://git.openjdk.org/jdk/pull/12441.diff</a>

</details>

### CODE REVIEW

File: test/jdk/com/sun/jdi/JdbLastErrorTest.java [hunk 1]
```diff
@@ -63,8 +63,9 @@ private static void testWindows() throws Throwable {
         for (int i = 0; i < 10; i++) {
             setLastError.invoke(VALUE);
             int lastError = (int) getLastError.invoke();
-            System.out.println(lastError);
+            System.out.println("lastError = " + lastError);
             if (lastError != VALUE) {
+                System.err.println("iteration " + i + " gets lastError = " + lastError);
```
Comment (dholmes-ora) [inline test/jdk/com/sun/jdi/JdbLastErrorTest.java:68]: s/gets/got/

**Review by dholmes-ora — APPROVED** [review]
Seems reasonable.

Thanks.

File: test/jdk/com/sun/jdi/JdbLastErrorTest.java (hunk 1 above)
Comment (sspitsyn) [inline test/jdk/com/sun/jdi/JdbLastErrorTest.java:68]: It'd be nice to print the expected VALUE as well.

**Review by sspitsyn — APPROVED** [review]
Looks good to me.
Posted one nit.
Thanks,
Serguei

**Review by plummercj — APPROVED** [review]

**Comment by kevinjwalls** [comment]
Thanks for the comments, will have the test logging for example:

[debuggee] iteration 0 got lastError = 0 (expected 42)

**Pull request closed without merging**